### ISSUE TITLE
suggestion: smart startup capability for NVDA

### ISSUE THREAD

**Issue by dave090679** [nvaccess/nvda#9586]
Hi guys,

instead of just having 2 Settings for the autostart option (on or off) we could have a third "smart" autostar option. If you selected the smart startup and you start nvda by pressing ctrl+alt+n, nvda should start and it should create an autostart shortcut for the current user. If the smart setting is enabled and you press nvda+q while nvda is running, the autostart shortcut should be deleted. 

The startup option should be presented as a combo box with 3 option: "always on", "always off" and "smart startup". 

the smart startup of nvda is mainly inspired by narrator on Windows 10, orca on Linux and Voiceover on the mac and on ios.

greetings

Dave

**Comment by DrSooom** [comment]
@dave090679: The hotkey CTRL+ALT+N is allocated to the NVDA's shortcut icon on the Windows Desktop. This is a Windows feature since many, many years. Even if NVDA isn't automatically started after logon, you still be able to launch it by pressing CTRL+ALT+N, unless you also delete that shortcut icon on the Windows Desktop. Therefore I can't see a benefit in this feature, as you're already able to disable NVDA startup after logon via the General NVDA Settings. Any other thoughts from the community?

**Comment by leonardder** [comment]
I understand this request and think that, though it might be a bit complex, could be valid.

**Comment by Brian1Gaff** [comment]
So are they saying they want nvda to only start on auto if no other 
screenreader is also on by default?
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by lukaszgo1** [comment]
I honestly don't see an advantages in implementing this. You are able to start NVDA at will by pressing Win+r and typing nvda even if desktop icon has no shortcut. At first I thought that it might be useful for shared computers like at the university but when blind user finishes working with it he usually shuts it down rather than closing NVDA AND turning it off blindly.

**Comment by Adriani90** [comment]
Here it seems there is a consensus that this would not bring higher benefits for users if having it implemented. I am closing this one. However, if you have a clear justification for this feature we can always consider reopening the discussion.
